### PR TITLE
fix capitalization issue for container name

### PR DESCRIPTION
**Pull request by tjmehta**

### CODE REVIEW

File: lib/middleware.js [hunk 1]
```diff
@@ -194,6 +197,7 @@ var middlewares = module.exports = {
       }
     };
     delete req.body.Labels;
+    var capitalizedBody = mapKeys(req.body, capitalize);
```
Comment (bkendall) [inline lib/middleware.js]: this var isn't used anywhere and is breaking lint

File: package.json [hunk 2]
```diff
@@ -30,6 +30,7 @@
     "keypather": "^1.10.1",
     "middleware-flow": "^0.8.0",
     "minimist": "^1.1.0",
+    "object-loops": "^0.2.0",
```
Comment (bkendall) [inline package.json]: oh man... can't wait for `for...of`

**Comment by bkendall** [comment]
Looks like Docker's API is much more consistent to this now too, to this is a Good Thing(tm). Thanks!

**Comment by anandkumarpatel** [comment]
LGTM

**Pull request merged**